src/backend/joanie/tests/swagger/admin-swagger.json

```diff
@@ -3947,13 +3947,19 @@
                     "uri": {
                         "type": "string",
                         "readOnly": true
+                    },
+                    "state": {
+                        "type": "string",
+                        "description": "Return the state of the course run at the current time.",
+                        "readOnly": true
```

Comment:
This is not reflecting the actual dict returned.
I got some kind of fix, let me know when you want to discuss it.

Reply:
Good catch, I followed your suggestion! 🙏